config/af.py

```diff
@@ -14,6 +14,14 @@
 
 NUMERALS = byte.DIGIT
 
+EMBIGGEN_CONTRACTIONS = cdrewrite(
```

Comment:
Haha, love the Simpsons reference

Reply:
It's a perfectly cromulent way to describe what this rule does